plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -391,6 +392,23 @@ class ServicesTable extends React.Component {
     );
   }
 
+  renderInstances(prop, service) {
+    const isFolder = service.isFolder();
```

Comment:
Please use the instance type to check wether it's a folder / `ServiceTree` or a plain `Service`.

Reply:
Can you take a look at this comment please https://github.com/dcos/dcos-ui/pull/2447#discussion_r143761703

Reply:
:)